Patches/RimTraits - General Traits/RT_General_Traits.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>RimTraits - General Traits</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<!-- ======= General Traits ======= -->
+
+				<!-- ===== RT Trait: Nearsighted ===== -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/TraitDef[defName = "RT_Nearsighted"]/degreeDatas/li/statOffsets</xpath>
+					<value>
+						<statOffsets>
+							<ShootingAccuracyPawn>-0.4</ShootingAccuracyPawn>
+							<AimingDelayFactor>0.15</AimingDelayFactor>
+							<MeleeHitChance>1</MeleeHitChance>
+							<MeleeDodgeChance>0.10</MeleeDodgeChance>
+						</statOffsets>
+					</value>
+				</li>
+
+				<!-- ===== RT Trait: Farsighted ===== -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/TraitDef[defName = "RT_Farsighted"]/degreeDatas/li/statOffsets</xpath>
+					<value>
+						<statOffsets>
+							<ShootingAccuracyPawn>0.5</ShootingAccuracyPawn>
+							<AimingDelayFactor>-0.15</AimingDelayFactor>
+							<MeleeHitChance>-1</MeleeHitChance>
+							<MeleeDodgeChance>-0.10</MeleeDodgeChance>
+						</statOffsets>
+					</value>
+				</li>
+
+				<!-- ===== RT Trait: Athletic ===== -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/TraitDef[defName = "RT_Athletic"]/degreeDatas/li/statOffsets/MeleeDodgeChance</xpath>
+					<value>
+						<MeleeDodgeChance>0.10</MeleeDodgeChance>
+					</value>
+				</li>
+
+				<!-- ===== RT Trait: Stiff ===== -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/TraitDef[defName = "RT_Stiff"]/degreeDatas/li/statOffsets/MeleeDodgeChance</xpath>
+					<value>
+						<MeleeDodgeChance>-0.10</MeleeDodgeChance>
+					</value>
+				</li>
+
+				<!-- ===== RT Trait: Muscular ===== -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/TraitDef[defName = "RT_Muscular"]/degreeDatas/li/statOffsets/CarryingCapacity</xpath>
+					<value>
+						<CarryWeight>2</CarryWeight>
```

Comment:
Unless this is a particularly weak trait, this can't probably be increased a bit. I'd give an increment of 5 weight/bulk.

Reply:
I agree, compared to StrongBack from Individuality, I think this one is a a bit weaker than that.